reasoner/runner/runner-common/src/main/java/com/antgroup/openspg/reasoner/kggraph/impl/KgGraphImpl.java

```diff
@@ -173,8 +173,33 @@ public List<KgGraph<IVertexId>> split(
       Long limit) {
     Set<String> vertexAliases = Sets.newHashSet(inputVertexAliases);
 
-    // remove alias that vertex set size is 1
-    vertexAliases.removeIf(vertexAlias -> 1 == this.alias2VertexMap.get(vertexAlias).size());
+    // remove alias that vertex set size is 1 and its related edge size is 1
+    Map<String, Set<String>> vertexAliasToEdgeAliasMap = new HashMap<>();
+    for (String key : JavaConversions.setAsJavaSet(schema.topology().keySet())) {
```

Comment:
Each invocation of the split function requires parsing the schema and obtaining the relevant edge information, which is too inefficient. This analysis process can be placed into KgGraphSplitStaticParameters.

Reply:
Changes have been made